contracts/Remittance.sol

```diff
@@ -84,12 +81,13 @@ contract Remittance is Stoppable {
         return true;
     }
     
-    function withdrawFees() public _onlyOwner returns(bool success) {
-        uint amount = fees[getOwner()];
+    function withdrawFees() public returns(bool success) {
+        address owner = getOwner();
+        uint amount = fees[owner];
```

Comment:
So only the current owner can withdraw?
If I am the former owner, and I still have something to withdraw, how do I do?